doc/get_started.rst

```diff
@@ -3,110 +3,99 @@
 Get started
 ===========
 
+Installation
+------------
 
-Install
--------
+The recommended way to use benchopt is within a conda environment to fully-benefit from all its features.
+Hence, start by creating a dedicated conda environment. 
 
-This package can be installed through `pip`.  In order to allow benchopt to automatically
-install solvers dependencies, the install needs to be done in a `conda` environment.
-
-.. prompt:: bash $
+.. code-block:: bash
 
+    # create conda environment with name benchopt
     conda create -n benchopt python
-    conda activate benchopt
-
-To get the **latest release**, use:
-
-.. prompt:: bash $
 
-    pip install benchopt
-
-And to get the **latest development version**, you can use:
+    # activate the environment
+    conda activate benchopt
 
-.. prompt:: bash $
+Benchopt is available on PyPi. You can get the **stable version** via ``pip`` by running the following command
 
-    pip install -U -i https://test.pypi.org/simple/ benchopt
+.. code-block:: bash
```

Comment:
Using `. prompt:: bash $` as before puts a dollar at the beginning of each line, making it clearer that it's a block of bash code. Can you revert?

Reply:
Also there's a nice feature where on the top right corner of the box there's a button that you can click to copy the snippet, if you manage to implement it :pray: https://sphinx-copybutton.readthedocs.io/en/latest/